test.py

```diff
@@ -1,27 +1,49 @@
+from sb.robot import *
```

Comment:
I always found when running the code that the actual import wasnt necessary, is that not correct anymore?

I also think importing `*` is a terrible anti-pattern that causes so many shadowing issues. Can we be explicit here

Reply:
That sounds slightly horrific. Maybe this is worth having another look into - at the least it'd be nice to document what this line is *actually* doing as opposed to what it appears to do.

Reply:
Not really, as the `test.py` code is intended for teams to copy and use as a basis for their own robots. I think it's best for them to just not know that this line does nothing.

Reply:
If this import isn't doing anything, we should remove it. I think encouraging people to use this code as a basis is probably a bad idea, and something they're _highly_ unlikely to do anyway

Semi-related, If we want them to use it as a basis, we should probably change it to `from robot import *`?

Reply:
Yes, will fix in separate PR as this feature hasn't been added in this one.